spec.html

```diff
@@ -130,21 +130,49 @@ <h1>ThrowAggregateError ( _errors_ )</h1>
   </emu-alg>
 </emu-clause>
 
+<emu-clause id="sec-well-known-intrinsic-objects">
+  <h1>Well-Known Intrinsic Objects</h1>
+  <emu-table id="table-7" caption="Well-Known Intrinsic Objects">
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Intrinsic Name
+          </th>
+          <th>
+            Global Name
+          </th>
+          <th>
+            ECMAScript Language Association
+          </th>
+        </tr>
+        <tr>
+          <td>
+            <ins>%AggregateError%</ins>
+          </td>
+          <td>
+            <ins>`AggregateError`</ins>
+          </td>
+          <td>
+            <ins>The `AggregateError` constructor (<emu-xref href="#sec-native-error-types-used-in-this-standard-aggregateerror"></emu-xref>)</ins>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </emu-table>
+</emu-clause>
+
 <emu-clause id="sec-native-error-types-used-in-this-standard-aggregateerror">
   <h1>AggregateError ( errors, message )</h1>
   <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
   <emu-alg>
     1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-    1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateErrorPrototype%"`, « [[ErrorData]] »).
-    1. Let _errorsIterator_ be ? GetMethod(_errors_, @@iterator).
-    1. Let _errorsList_ ? IterableToList(_errors_, _errorsIterator_).
-    1. Let _errorsArray_ be CreateArrayFromList(_errorsList_).
-    1. Let _errorsDesc_ be the PropertyDescriptor { [[Value]]: _errorsArray_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
-    1. Perform ! DefinePropertyOrThrow ( _O_, `"errors"`, _errorsDesc_ ).
+    1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+    1. Let _errorsArray_ be ! CreateArrayFromList(? IterableToList(_errors_)).
```

Comment:
I am not a fan of using multiple `!`/`?` macros in a single algorithm step. It's probably clear to a human, but it is [not allowed by the current definition of the macros](https://tc39.es/ecma262/#sec-returnifabrupt) and it is not totally clear to me if it would be allowed after https://github.com/tc39/ecma262/pull/1573.

Reply:
We definitely do it in the spec in a number of places already; if 1573 wouldn’t allow it, we should ensure it will.

I could split it up, but I don’t think it adds clarity to do so.

Reply:
I tend to agree with:

> (And there's no real reason not to just introduce a new variable and make it totally unambiguous.)

Happy to make this change separately though. No need to block this pull request.